.travis/travis-release.sh

```diff
@@ -44,6 +44,22 @@ if [[ -z $TRAVIS_TAG ]] && [[ ! $TRAVIS_COMMIT_MESSAGE =~ $COMMIT_MESSAGE_PATTER
   eval $(ssh-agent -s)
   ssh-add deploy_rsa
 
+  # build charts/images and push
+  cd helm-chart
+  chartpress --push --publish-chart
+  git diff
+  # push also images tagged with "latest"
+  chartpress --tag latest --push
+elif [[ -z $TRAVIS_TAG ]] && [[ $TRAVIS_COMMIT_MESSAGE =~ $COMMIT_MESSAGE_PATTERN ]]; then
+  # push the charts and images
```

Comment:
If I see this correctly, the entire code below is just copy-pasted from above. Please avoid the duplication as it will make it much easier to see what the difference in the handling of the two cases is.

Reply:
No cutting corners! I thought no one will catch that :)

Reply:
All the commonalities moved to a separate script file.